samples/Umbraco.Headless.Client.Samples.Console/Umbraco.Headless.Client.Samples.Console/Program.cs

```diff
@@ -145,6 +148,69 @@ async Task ListAllContentUrls(ContentDeliveryService service)
     Console.WriteLine(" ");
 }
 
+async Task CreateContentWithMediaReference(ContentDeliveryService contentDeliveryService, string projectAlias)
+{
+    Console.WriteLine(" ");
+    Console.WriteLine("In order to create content you need to authenticate against the Umbraco Headless Backoffice");
+
+    Console.WriteLine(" ");
+    Console.WriteLine("Enter your api key:");
+    var apiKey = PromptUser("API Key", true);
+    if (apiKey.Equals("exit", StringComparison.OrdinalIgnoreCase)) return;
+
+    var managementService = new ContentManagementService(projectAlias, apiKey);
+
+    Console.WriteLine(" ");
+    Console.WriteLine("Enter the content type alias of the content you want to create:");
+    var contentTypeAlias = PromptUser("content type alias");
+    if (contentTypeAlias.Equals("exit", StringComparison.OrdinalIgnoreCase)) return;
+
+    Console.WriteLine(" ");
+    Console.WriteLine("Enter the property alias of the media picker property you want add an image to:");
+    var mediaPickerPropertyAlias = PromptUser("media picker property alias");
+    if (mediaPickerPropertyAlias.Equals("exit", StringComparison.OrdinalIgnoreCase)) return;
+
+    Console.WriteLine(" ");
+    Console.WriteLine("Enter the ID of the media item you want to reference:");
+    var mediaId = PromptUser("media ID");
```

Comment:
Given this reference and the mention of using `UploadImageToMedia` prior to this option I think we should ensure that the `UploadImageToMedia` also prints out the id of the media to the console. Right now it only prints out the name and url of the media item.

Reply:
Good idea! I have added printing the id to the `RenderMediaWithUrl` method.